src/types.ts

```diff
@@ -100,6 +100,10 @@ export type BaseToastOptions = {
    */
   from?: "top" | "bottom";
   layout?: Layout;
+  action?: {
```

Comment:
Can you add a platform comment here

Reply:
Done!

Reply:
It’s resolved but the new code doesn’t show as pushed on my end